OSDOCS-12321-RN#Adding WIF storage support (#89135)
openshift/openshift-docs: release_notes/ocp-4-17-release-notes.adoc

$ oc adm release info 4.17.11 --pullspecs
----

[id="ocp-4-17-11-enhancements_{context}"]
==== Enhancements

[id="ocp-4-17-11-wif-storage-support_{context}"]
===== GCP Filestore supporting Workload Identity is generally available

> Google Compute Platform (GCP) Filestore Container Storage Interface (CSI) storage supports Workload Identity. This allows users to access Google Cloud resources using federated identities instead of a service account key. This feature is generally available.

Reviewer: ```suggestion
Google Compute Platform (GCP) Filestore Container Storage Interface (CSI) storage supports Workload Identity. You can now access Google Cloud resources by using federated identities instead of a service account key. This feature is generally available.
```

Avoid using "allow": https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a

> Avoid stating that inanimate objects grant abilities to people, as in “the product allows you to…”. Whenever possible, use a direct, user-focused alternative such as “you can use the product to…” or “with this product, users can…”.
